static/css/legacy.less

```diff
@@ -535,6 +535,7 @@ input {
 
 // openlibrary/templates/work_search.html
 .advanced-search {
+  font-size: 12px;
```

Comment:
Would it be a terrible thing to drop this style?

Reply:
I'll try dropping it on staging to see if it makes a difference on appearance.

Reply:
Ooooh, looks very weird 😬 Let's leave this for now and clean this up later.

![image](https://user-images.githubusercontent.com/6251786/118297431-21d65000-b4ac-11eb-80b1-a9597c2ac3b2.png)